tests/src/Functional/SubsiteBlocksTest.php

```diff
@@ -233,6 +233,17 @@ public function testSubsiteNavigationBlock() {
     $this->assertSession()->responseNotContains('block-localgov-subsite-navigation');
     $this->drupalGet($subsite_page1->toUrl()->toString());
     $this->assertSession()->responseNotContains('block-localgov-subsite-navigation');
+
+    // Test the navigation is still avalible with an unpublished overview node.
+    $subsite_overview->set('localgov_subsites_hide_menu', ['value' => 0]);
+    $subsite_overview->set('status', NodeInterface::NOT_PUBLISHED);
+    $subsite_overview->save();
+    drupal_flush_all_caches();
```

Comment:
So my question is, is it ok to add a `drupal_flush_all_cache` here. Assuming this is what caused the cache break as the cache tags arn't being expired since its not saving the form?

Reply:
I tried clearing the node cache for the nodes involved and the render cache and that didn't fix things, so I think it's fine.